portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,35 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.speech-bubble {
+  background-color: #f8f8f8;
+  border: 2px solid #c8c8c8;
+  border-radius: 10px;
+  width: 500px;
```

Comment:
nit: General CSS practice is to alphabetize properties

Reply:
Thanks! I'll keep that in mind in the future.